docs/why-logfire/index.md

```diff
@@ -0,0 +1,59 @@
+# Introducing Pydantic Logfire
+
+From the team behind Pydantic, **Logfire** is an observability platform built on the same belief as our open source library — that the most powerful tools can be easy to use.
+
+## What sets Logfire apart
+
+<div class="grid cards" markdown>
+
+-   :rocket:{ .lg .middle } __Simplicity and Power__
+
+    ---
+
+    Logfire's dashboard is simple relative to the power it provides, ensuring your entire engineering team will actually use it. Time-to-first-log should be less than 5 minutes.
+
+    [:octicons-arrow-right-24: Read more](simplicity.md)
+
+-   :snake:{ .lg .middle } __Python-centric Insights__
+
+    ---
+
+    From rich display of **Python objects**, to **event-loop telemetry**, to **profiling Python code &amp; database queries**, Logfire gives you unparalleled visibility into your Python application's behavior.
+
+    [:octicons-arrow-right-24: Read more](python-centric.md)
+
+-   :simple-pydantic:{ .lg .middle } __Pydantic Integration__
+
+    ---
+
+    Understand the data flowing through your Pydantic models and get built-in analytics on validations.
+
+    Pydantic Logfire helps you instrument your applications with less code, less time, and better understanding.
+
+    [:octicons-arrow-right-24: Read more](pydantic.md)
+
+-   :telescope:{ .lg .middle } __OpenTelemetry__
+
+    ---
+
+    Logfire is an opinionated wrapper around OpenTelemetry, allowing you to leverage existing tooling, infrastructure, and instrumentation for many common Python packages, and enabling support for virtually any language.
+
+    [:octicons-arrow-right-24: Read more](opentelemetry.md)
+
+-   :abacus:{ .lg .middle } __Structured Data and SQL__
```

Comment:
it looks a bit weird to have 5 when they're laid out in a 2-grid.

Could we split "Structured Data" and "SQL" into to panels, even if for now they link to the same page?

Reply:
This is done - I just took the copy from the page
![Screenshot 2024-09-24 at 11 53 50](https://github.com/user-attachments/assets/c812a6b8-ab1a-4ddd-81fa-cf5aa0251802)